### DISCUSSION TITLE
"Get last file name" reporter

### DISCUSSION THREAD

**Discussion by snowboyz0825** [TurboWarp/extensions#1428] [Ideas]
It would be cool to have a block in the files extension that allows you to get the name of the last file uploaded. I would write more but there's not much more to be said, it would _probably_ behave like the answer block, but instead of reporting the last answer given reporter the name of the last file loaded.

**Comment by SharkPool-SP** [comment]
> It would be cool to have a block in the files extension that allows you to get the name of the last file uploaded. I would write more but there's not much more to be said, it would _probably_ behave like the answer block, but instead of reporting the last answer given reporter the name of the last file loaded. 

You should use Files Expanded, found on my gallery, it has that block and more!!

https://sharkpool-sp.github.io/SharkPools-Extensions/

**Reply by AshimeeAlt** [reply]
bro is even advertising on github now :skull:

**Reply by SharkPool-SP** [reply]
> bro is even advertising on github now :skull: 

Now? I've been doing this for a while :trollface:
